bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/bubble/Bubble.java

```diff
@@ -178,12 +183,13 @@ public Bubble(final double xPos, final double yPos, final ColorChoice c,
     }
   }
 
+  public Bubble() {}
+
   /**
    * Draws the Bubble.
    * @param g  Java Graphics instance
    */
-  // Might have abstract or final tag.
-  public final void draw(final Graphics g) {
+  public void draw(final Graphics g) {
```

Comment:
So I'm not getting a checkstyle error - not sure how to avoid this.

`draw` is overridden in Decorator classes (thus can not be final), and `draw` in Bubble itself has a meaningful implementation (thus not abstract).

I think the checkstyle error is resolved here since I provide an override in subclasses